doc/source/ray-air/examples/torch_image_example.ipynb

```diff
@@ -564,6 +568,8 @@
                 "    dtype=torch.float,\n",
                 "    feature_columns=[\"image\"],\n",
                 "    keep_columns=[\"label\"],\n",
+                "    # We will use GPU if available.\n",
+                "    num_gpus_per_worker=ray.available_resources().get(\"GPU\", 0) > 0\n",
```

Comment:
Should this be?
```suggestion
                "    num_gpus_per_worker=ray.available_resources().get(\"GPU\", 0)\n",
```

Or should we be more explicit to convert to 1 if there are GPU available?